packages/cli/src/zosjobs/delete/job/Job.definition.ts

```diff
@@ -30,6 +30,15 @@ export const JobDefinition: ICommandDefinition = {
             required: true
         }
     ],
+    options: [
+        {
+            name: "modifyVersion",
+            description: "If you use this option, X-IBM-Job-Modify-Version will be set to \"2.0\" and delete job API will be synchronous. " +
+            "Otherwise, it will be asynchronous.",
+            type: "boolean",
+            required: false
+        }
```

Comment:
Regarding the CLI command option `--modifyVersion`, would it make sense to define it as an integer with `allowableValues` of 1 and 2? I'm not sure how futureproof it is to make it a boolean, in case a version 3 is created in the future.

Reply:
I'm in agreement with Timothy here. While I understand that the main difference right now is whether or not the operation is asynchronous, it is defined in [IBM documentation](https://www.ibm.com/docs/en/zos/2.4.0?topic=interface-cancel-job-purge-its-output) as a string (specifically, "1.0" and "2.0"). If those headers were "true" and "false", I would be more open to changing the type of this argument on the next branch.